js/fr.js

```diff
@@ -0,0 +1,147 @@
+var refFr = {"site-map": {
+	"LOCATING_FAILURE": "N'a pas trouvé votre lieu",
+	"LOCATING_SUCCESS": "Votre lieu",
+	"LOADING_FAILURE": "Une erreur s'est produite, veuillez zoomer et réessayer",
+	"LNK_OSM_EDIT": "Modifier dans OSM",
+	"LNK_OSM_REPORT": "Signaler une information erronée",
+	"LNK_OSM_VIEW": "Voir le Point d'intérêt dans OpenStreetMap",
+	"LNK_OPEN_WITH": "Ouvrir avec une appli",
+	"FLTR_SELECTALL": "(De)sélectionnez tout",
+	"PDV_UNNAME": "Sans Nom",
+	"PDV_ADDRESS_UNKNOWN": "Addresse inconnue",
+	"PDV_COMMUNE_UNKNOWN": "?",
+	"PDV_STREET_UNKNOWN": "Rue inconnue",
+	"PDV_HOUSENUMBER_UNKNOWN": "?",
+	"PDV_ZIPCODE_UNKNOWN": "Code postal inconnu",
+	"PDV_TITLE_HOME": "General",
+	"PDV_TITLE_BABY": "Baby friendliness",
```

Comment:
Yes I have skipped this line because it is difficult to translate.

There is no translation in French for the word _friendly_ so according of the context we do a periphrasis (for example _Adapté aux bébés_ which is in fact the translation of _Suitable for babies_) or we let the word _friendly_ (for example _Bébé friendly_).

I will search the context to chose the best option

Reply:
Context : 
![image](https://user-images.githubusercontent.com/13560473/60284222-4ef6d500-990b-11e9-8bf9-a5d39b61790e.png)

The menu show in fact the existence or not of equipment. So I will translate _Baby friendliness_ by _Équipements pour bébés_ .

Reply:
Done in 66321d7

Reply:
The stroller key is not about equipment. So I think Adapté aux bébés might actually be the better translation.

Reply:
You're right, I have fixed it in af8e38c